.github/workflows/codeql.yml

```diff
@@ -13,7 +13,7 @@ name: 'CodeQL'
 
 on:
   push:
-    branches: ['main', archive/*, main*, release-*, release/*]
+    branches: ['main', archive/*, main*, release-1\.(1[3-9]|[2-9]\d|\d{3,})\.+.*, release/*]
```

Comment:
What do you think about adding a comment here to explain the regex? Personally I have to do some extra work to see what it means, and even if I figured it out, I don't think I would understand why we do 1.13.* and higher until I dug my way to the PR

Reply:
No problem